spec/page_spec.rb

```diff
@@ -180,31 +191,42 @@ class PageWithBogusFullUrlMatcher < SitePrism::Page
     specify '#url_matches raises InvalidUrlMatcher' do
       expect { page.url_matches }
         .to raise_error(SitePrism::InvalidUrlMatcher)
-        .with_message('Could not automatically match your URL. Templated port numbers are unsupported.')
+        .with_message(
+          "Could not automatically match your URL. \
+Templated port numbers are unsupported."
+        )
     end
 
     specify '#displayed? raises InvalidUrlMatcher' do
       expect { page.displayed? }
         .to raise_error(SitePrism::InvalidUrlMatcher)
-        .with_message('Could not automatically match your URL. Templated port numbers are unsupported.')
+        .with_message(
+          "Could not automatically match your URL. \
+Templated port numbers are unsupported."
+        )
     end
   end
 
   describe 'with a full string URL matcher' do
     class PageWithStringFullUrlMatcher < SitePrism::Page
-      set_url_matcher 'https://joe:bump@bla.org:443/foo?bar=baz&bar=boof#myfragment'
+      set_url_matcher \
```

Comment:
can we revert this one for now. I don't like how we're already stretching the line before defining anything

Reply:
:point_up: